docs/containers-transports.5.md

```diff
@@ -0,0 +1,53 @@
+% CONTAINERS-TRANSPORTS(5) Containers Transports Man Page
+% Valentin Rothberg
+% April 2019
+
+# NAME
+containers-transports - description of supported transports for copying container images
+
+# DESCRIPTION
+
+**containers-storage:**_docker-reference_
+
+An image located in a local containers storage image store.
+Please refer to containers-storage.conf(5) for further information.
+
+**dir:**_path_
+
+An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files.
+This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+**docker://**_docker-reference_
+
+An image in a registry implementing the "Docker Registry HTTP API V2".
+By default, uses the authorization state in either `$XDG_RUNTIME_DIR/containers/auth.json`, which is set using `(podman login)`.
+If the authorization state is not found there, `$HOME/.docker/config.json` is checked, which is set using `(docker login)`.
+
+**docker-archive:**_path_[**:**_docker-reference_]
+
+An image is stored in the `docker save` formatted file.
+_docker-reference_ is only used when creating such a file, and it must not contain a digest.
+Note that it is possible to copy data to stdin by specifying `docker-archive:/dev/stdin`.
+
+**docker-daemon:**_docker-reference_
+
+An image _docker-reference_ stored in the docker daemon internal storage.
```

Comment:
"A _docker-reference_ formatted image stored" perhaps?

Reply:
The image is not “reference-formatted”